.vscode/launch.json

```diff
@@ -347,9 +347,10 @@
 			"request": "launch",
 			"name": "Debug Current Mocha Test (auto build)",
 			"env": {
-				// "fluid__test__driver": "odsp", //values: local, tinylicious, routerlicious, odsp,
-				// "fluid__test__backCompat": "FULL", //values: FULL This tests loader-driver compatibility for describeCompat tests
+				// "fluid__test__driver": "odsp", //values: local, t9s, r11s, odsp,
 				// "fluid__test__odspEndpointName": "odsp-df", //values: odsp, odsp-df
+				// "fluid__test__r11sEndpointName": "docker", //values: frs, docker
+				// "fluid__test__backCompat": "FULL", //values: FULL This tests loader-driver compatibility for describeCompat tests
 				// "FLUID_TEST_VERBOSE": "1",
```

Comment:
Part of me wants to leave this uncommented, I normally just uncomment this in my branches. Not sure if that's a good idea, but food for thought.

Everything else looks good